_gsocprojects/2021/project_MadGraph5.md

```diff
@@ -0,0 +1,10 @@
+---
+project: MadGraph5
+title: MadGraph5_aMC@NLO
+layout: default
+logo: MadGraph5-logo.png
+description: |
+  [MadGraph5_aMC@NLO](https://launchpad.net/mg5amcnlo) is a framework that aims to simulate high Energy collisions for various accelerators and in particular for LHC physics. It has a huge focus on physics beyond the “Standard Model” allowing experiments to search for various scenarios in their experiments. MadGraph5 is highly used by both theorists and experimentalists making the code one of the most cited papers every year. MadGraph5 is a meta-code written in python that generates specialised code in low-level language (Fortran and C) to produce very fast code.
```

Comment:
"high Energy" -> inconsistency in capitalization, maybe "high-energy"
"huge" -> "clear"/"well-defined" (?) - just a suggestion
"experiments to search ... in their experiments" -> maybe replace first "experiments" with "researchers" or "scientists" to avoid repetition
"highly used" -> "extensively used"
"in low-level language" -> "in a low-level language" or "in low level languages"

Reply:
thanks for spotting, I took your suggestions or something close